### DISCUSSION TITLE
机器人好像不挪掉部分标签

### DISCUSSION THREAD

**Discussion by MoYuan-CN** [Meloong-Git/PCL#2774] [提问] [answered]
#477 帖的 [这个链接](https://github.com/Hex-Dragon/PCL2/issues?q=is%3Aissue+label%3A%E6%96%B0%E5%8F%8D%E9%A6%88%2C%E9%9C%80%E8%A6%81%E7%A4%BE%E5%8C%BA%E5%A4%8D%E7%8E%B0%2C%E9%9C%80%E8%A6%81%E7%A4%BE%E5%8C%BA%E5%B8%AE%E5%8A%A9%2C%E7%AD%89%E5%BE%85%E5%8F%8D%E9%A6%88%E8%80%85+) 点击过去发现一个真正要处理的都没 全是关了的
![image](https://github.com/Hex-Dragon/PCL2/assets/109946426/575332ce-fc89-4298-980d-65ef20f8aca6)

**Comment by WForst-Breeze** [comment, marked as answer]
这些 issue 就是因为需要社区帮助才会被关闭的，当相关信息完备后会再次打开

**Reply by MoYuan-CN** [reply]
o
